index.html

```diff
@@ -291,7 +301,18 @@ <h3>What is a Verifiable Credential?</h3>
         </p>
 
         <p>
-<a>Holders</a> of <a>verifiable credentials</a> can generate
+There are 2 broad categories of <a>verifiable credentials</a>: <a>bound credentials</a> and <a>unbound credentials</a>. 
```

Comment:
I'm concerned about adding new classes of verifiable credential. We tried hard to reduce the amount of new terminology that people would need to learn to use VCs and I'm not sure the addition of this distinction is helpful. I think it might be harmful.

That is not to say that the concept doesn't exist. It does, and we should document it. I just don't think we should give it a misleading name.

Reply:
> I'm concerned about adding new classes of verifiable credential. We tried hard to reduce the amount of new terminology that people would need to learn to use VCs and I'm not sure the addition of this distinction is helpful. I think it might be harmful.
> 
> That is not to say that the concept doesn't exist. It does, and we should document it. I just don't think we should give it a misleading name.

@msporny I think you're missing a key point in this discussion: @swcurran and I have stumbled on the need/requirement for a more formal taxonomic approach to describing the different species and subspicies of credentials. After you learn more about these concepts, I think you'll #seethelight (aka #seetheproof).